editoast/src/railjson/operation/update.rs

```diff
@@ -186,4 +205,115 @@ mod test {
             Ok(())
         });
     }
+
+    #[test]
+    fn valide_update_signal() {
+        let conn = PgConnection::establish(&PostgresConfig::default().url()).unwrap();
+        conn.test_transaction::<_, Error, _>(|| {
+            let signal_creation = RailjsonObject::Signal {
+                railjson: Signal {
+                    ..Default::default()
+                },
+            };
+            let infra = Infra::create(&"test".to_string(), &conn).unwrap();
+            assert!(apply_create_operation(&signal_creation, infra.id, &conn).is_ok());
+
+            let update_signal = UpdateOperation {
+                obj_id: "my_signal".to_string(),
+                obj_type: ObjectType::Signal,
+                railjson_patch: from_str(
+                    r#"[
+                    { "op": "replace", "path": "/sight_distance", "value": 15.0 }
+                  ]"#,
```

Comment:
Just a note, we're using `from_str` multiple times with a static string.
Instead we could use `json!` macro. I think it will be more elegant but I'm not sure.